interpreter/misc_test.go

```diff
@@ -39,316 +39,30 @@ import (
 	"github.com/onflow/cadence/runtime"
 	"github.com/onflow/cadence/sema"
 	"github.com/onflow/cadence/stdlib"
+	"github.com/onflow/cadence/test_utils"
```

Comment:
Maybe make this a dot-import, like with the other test utils packages

Reply:
Not possible yet, because some duplicate types are also defined in this file (`misc_test.go`) as well. Would need to refactor all tests that uses them and then we can safely remove the duplicates from here. Will do that in a separate PR.